.travis.yml

```diff
@@ -2,7 +2,7 @@ language: minimal
 
 env:
   global:
-    - VERSION=0.1.2
```

Comment:
version number in the packages are based on the $VERSION env var. this is also hardcoded in the meson.build file. we figure out a single source of truth for this so this value can be derived everywhere else

Reply:
This sounds like a reasonable first approach. There is another approach based off of tags that ctypesgen is using. I don't completely understand it, but it seems reasonable for a next iteration on how to do this...

Reply:
I don't completely understand what the CMA package is supposed to do...
I originally didn't think we needed a CMA package, but I decided I was wrong. I decided that it would have to consist of the systemd config to start/stop/control/monitor the CMA docker container. I haven't thought much about it since then. Might also have to have systemd config to start/stop/manage the Neo4j container. But these changes seem reasonable for the moment.